examples/docking-protein-ligand-shape/docking-protein-ligand-shape.cfg

```diff
@@ -0,0 +1,91 @@
+# ====================================================================
+# Protein-ligand docking example
+
+# directory in which the scoring will be done
+run_dir = "run1"
+ncores = 20
+# cns_exec = "path/to/bin/cns" # optional
+
+# molecules to be docked
+molecules =  [
+    "data/3DV1-fit-prot.pdb",
+    "data/ligand-ensemble.pdb",
+    "data/shape.pdb"
+    ]
+
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+ligand_param_fname = "data/ligand.param"
+ligand_top_fname = "data/ligand.top"
+delenph = false
+autohis = false
+[topoaa.input.mol1]
+whatever_new_parameter_here=false
```

Comment:
Think you wish to remove this line :relaxed:

Reply:
Ok - I thought I was following your instructions ;-)